packages/web-sdk/src/transports/otlp/transform/LogLogRecord.test.ts

```diff
@@ -0,0 +1,64 @@
+import { LogEvent, LogLevel, TransportItem, TransportItemType } from '@grafana/faro-core';
+import { LogLogRecord } from './LogLogRecord';
+
+const item: TransportItem<LogEvent> = {
```

Comment:
If we define them outside of a `beforeEach`, it's best to declare them as constant objects:
```ts
const example = {
  // ... props
} as const;
```